src/config/keys.ts

```diff
@@ -191,6 +191,27 @@ export const buildActionsKey = (args: {
   },
 ];
 
+export const buildAggregateActionsKey = (args: {
+  itemId: UUID;
+  view: string;
+  requestedSampleSize: number;
+  type: string;
+  countGroupBy: string[];
+  aggregateFunction: string;
+  aggregateMetric: string;
+  aggregateBy: string[];
+}) => [
+  'aggregateActions',
+  args.itemId,
+  args.view,
+  args.requestedSampleSize,
+  args.type,
+  args.countGroupBy,
+  args.aggregateFunction,
+  args.aggregateMetric,
+  args.requestedSampleSize,
```

Comment:
Maybe you could use the object directly as second argument. React-query allows to use objects as keys [react-query docs](https://tanstack.com/query/v3/docs/react/guides/query-keys)